src/core/font_substitutions.js

```diff
@@ -0,0 +1,271 @@
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-Bold",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Bold",
+      },
+      path: "LiberationSans-Bold.ttf",
+      style: {
+        style: "normal",
+        weight: "bold",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-Oblique",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Italic",
+      },
+      path: "LiberationSans-Italic.ttf",
+      style: {
+        style: "italic",
+        weight: "normal",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Helvetica-BoldOblique",
+    {
+      local: {
+        alias: "Helvetica",
+        append: " Bold Italic",
+      },
+      path: "LiberationSans-BoldItalic.ttf",
+      style: {
+        style: "itealic",
+        weight: "bold",
+      },
+      ultimate: "sans-serif",
+      baseName: "helvetica",
+    },
+  ],
+  [
+    "Courier",
+    {
+      local: [
+        "Courier",
+        "Courier New",
+        "Liberation Mono",
+        "Nimbus Mono",
+        "Nimbus Mono L",
+        "Cousine",
+        "Cumberland",
+        "TeX Gyre Cursor",
+        "FreeMono",
+      ],
+      style: {
+        style: "normal",
+        weight: "normal",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-Bold",
+    {
+      local: {
+        alias: "Courier",
+        append: " Bold",
+      },
+      style: {
+        style: "normal",
+        weight: "bold",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-Oblique",
+    {
+      local: {
+        alias: "Courier",
+        append: " Italic",
+      },
+      style: {
+        style: "italic",
+        weight: "normal",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+  [
+    "Courier-BoldOblique",
+    {
+      local: {
+        alias: "Courier",
+        append: " Bold Italic",
+      },
+      style: {
+        style: "italic",
+        weight: "bold",
+      },
+      ultimate: "monospace",
+      baseName: "courier",
+    },
+  ],
+]);
+
+function makeLocal(local) {
+  let append = "";
+  if (!Array.isArray(local)) {
+    append = local.append;
+    local = substitutionMap.get(local.alias).local;
+  }
+  return local.map(name => `local(${name}${append})`).join(",");
+}
+
+function getFontSubstitution(docId, basePath, name) {
+  const substitution = substitutionMap.get(name);
+  if (!substitution) {
+    return null;
+  }
+
+  const { local, path, style, ultimate, baseName } = substitution;
+  const loadedName = `${docId}_${baseName}`;
+  let src = makeLocal(local);
+  if (path && basePath !== null) {
+    if (basePath.endsWith("/")) {
+      src += `,url(${basePath}${path})`;
+    } else {
+      src += `,url(${basePath}/${path})`;
+    }
```

Comment:
Please note that the API-option is documented as *requiring* a trailing slash, [please see here](https://github.com/mozilla/pdf.js/blob/797f8d3dca5e0770c83788b3b65387884031caa0/src/display/api.js#L185-L186), hence please simplify this as:
```suggestion
    src += `,url(${basePath}${path})`;
```